.nuspec/Xamarin.Forms.targets

```diff
@@ -92,4 +93,23 @@
 			DebugType = "$(DebugType)"
 			KeepXamlResources = "$(XFKeepXamlResources)" />
 	</Target>
-</Project>
+
+	<!-- CssG -->
+		<PropertyGroup>
+		<CoreCompileDependsOn>
+			CssG;
+			$(CoreCompileDependsOn);
+		</CoreCompileDependsOn>
+	</PropertyGroup>
+
+	<Target Name="CssG">
```

Comment:
the new CssG task is responsible for creating XamlResourceIdAttributes for Css file